src/Controls/src/Core/DatePicker.cs

```diff
@@ -39,11 +39,14 @@ public DatePicker()
 		{
 			_platformConfigurationRegistry = new Lazy<PlatformConfigurationRegistry<DatePicker>>(() => new PlatformConfigurationRegistry<DatePicker>(this));
 		}
-
-		public DateTime Date
+	
+		public DateTime? Date
 		{
 			get { return (DateTime)GetValue(DateProperty); }
-			set { SetValue(DateProperty, value); }
+			set { 
+				SetValue(DateProperty, value);
+				SetValue(DateProperty, value);
```

Comment:
We only need one here.

Reply:
@mattleibow removed white spaces as requested and updated branch please check.